examples/simple/README.md

```diff
@@ -0,0 +1,45 @@
+# Minimal working example
+
+The commands in `sh commands.sh` illustrate WExT on a small toy example.  If the dependencies for WExT are correctly installed, then the `commands.sh` script should run in seconds on any modern computer.  These commands have three major parts.
+
+1. Process mutation data (`process_mutations.py`)
+2. Compute mutation weights/probabilities (`compute_mutation_probabilities.py`).
+3. Find gene sets with significant patterns of mutually exclusive or co-occurring mutations (`find_sets.py`).
+
+This example applies WR-exact tests for mutual exclusivity and co-occurrence on pairs of genes in a small toy dataset using a saddlepoint approximation of each test.
+
+## Input
+
+The dataset has 8 genes, `a`, `b`, ..., `h`, and 14 patients, `1`, `2`, ..., `14`, where all mutations in genes `a` and `b` are mutually exclusive and all mutations in genes `c` and `d` are co-occurring.
+```
+1	a	c	d	e	f
+2	b	c	d	g	h
+3	a	c	d
+4	b	c	d
+5	a	c	d
+6	b	c	d
+7	a	c	d
+8	b
+9	a
+10	b
+11	a	e
+12	b	g
+13	c   d
+14  c   d
+```
+
+## Output
+
+The mutual exclusivity results (`excusivity_results-k2.tsv`) should be very similar to the following table, which shows that only genes `a` and `b` have mutually exclusive mutations with the default FDR < 0.5.
+```
+#Gene set	WRE (Saddlepoint) P-value	WRE (Saddlepoint) FDR	WRE (Saddlepoint) Runtime	T	Z	t00	t01	t10	t11
+a, b	0.000227547596931	0.0113258255022	0.00342392921448	12	0	2	6	6	0
+```
+
+The co-occurrence results (`any-co-occurrence_results-k2.tsv` and `all-co-occurrence_results-k2.tsv`) should be very similar to the following table, which shows that only genes `c` and `c` have co-occurring mutations with the default FDR < 0.5.
+```
+#Gene set	WRE (Saddlepoint) P-value	WRE (Saddlepoint) FDR	WRE (Saddlepoint) Runtime	T	Z	t00	t01	t10	t11
+c, d	0.00351621696959	0.175014196425	0.00170493125916	0	9	5	0	0	9
+```
+
+The WExT wiki describes the details of the above tests and the arguments for the above scripts in more detail.
```

Comment:
Maybe add a link to WExT wiki?

Reply:
Done, including separate links for each script.